src/kili/services/export/types.py

```diff
@@ -21,3 +21,29 @@ class JobCategory(NamedTuple):
 
 
 YoloAnnotation = Tuple[int, float, float, float, float]
+
+
+InputType = Literal["IMAGE", "TEXT"]
+MLTask = Literal["CLASSIFICATION", "NAMED_ENTITIES_RECOGNITION", "OBJECT_DETECTION"]
+Tool = Literal["rectangle", "semantic", "polygon"]
+JobName = NewType("JobName", str)
+ProjectId = NewType("ProjectId", str)
```

Comment:
We add a lot of duplicate types like these. We should put the most frequent one like these at the root of the package

Reply:
moved to `services/types.py`